### DISCUSSION TITLE
页眉格式

### DISCUSSION THREAD

**Discussion by wangyizhuo777** [stone-zeng/fduthesis#332] [Q&A] [answered]
又来麻烦大佬了。院系这次格式要求页眉格式，请问怎么把页眉设置为只显示“第X章  XXXX”，不显示小章节，同时居中显示呢？目前用的时twoside模式

**Comment by sikouhjw** [comment]
哪有小章节？

![@M~JQG2C5($CI_N6YUEYL](https://github.com/stone-zeng/fduthesis/assets/49770925/001c6853-b3b2-495e-bb8c-1c30dcc5c1d8)

**Reply by wangyizhuo777** [reply]
![1](https://github.com/stone-zeng/fduthesis/assets/163975097/03601b84-e56b-4fa4-a56e-216afae69d2d)
![2](https://github.com/stone-zeng/fduthesis/assets/163975097/c14c3cc0-5201-45d5-8f2e-571ebc5175fc)
目前显示是这样，偶数页显示章节，奇数页显示小章节，能否所有页都只显示章节呢？

**Comment by sikouhjw** [comment, marked as answer]
https://github.com/stone-zeng/fduthesis/blob/d84e0dd7df657d47943e98c98ff843cc1336bc5f/source/fduthesis.dtx#L3792-L3802
设置成 `C` 居中，并且全部换成 `\leftmark`

**Reply by wangyizhuo777** [reply]
牛的
